modules/appeals_api/app/swagger/appeals_api/v2/api_description.md

```diff
@@ -7,6 +7,9 @@ The Decision Reviews API passes data through to Caseflow, a case management syst
 
 Because this application is designed to allow third-parties to request information on behalf of a Veteran, we are not using VA Authentication Federation Infrastructure (VAAFI) headers or Single Sign On External (SSOe).
 
+### Character Support
+<p>Our CREATE endpoints support the Windows-1252 character set.</p>
```

Comment:
Didn't recognize this lat time, but this is a markdown file - you can add newlines instead of resorting to wrapping things in `<p>` tags.